src/config.ts

```diff
@@ -77,5 +81,11 @@ export function parseConfig(opts: Partial<Config>): Config {
     discardOldKeys: coalesce(opts.discardOldKeys, false),
     jsonSpace: coalesce(opts.jsonSpace, 2),
     enableExperimentalIcu: coalesce(opts.enableExperimentalIcu, false),
+    transComponentNames: coalesce(opts.transComponentNames, [
+      {
+        sourceModule: 'react-i18next',
+        importName: 'Trans',
```

Comment:
I wonder how much we want `Trans` component to be part of the default value. On the one hand it allows the user to disable raw `Trans` component extraction straight away which is good, but on the other hand it forces the user to specify the default value again if they just want to extend it. I'd like to hear your rationale.

Reply:
My rationale is my usecase, which is: overriding the Trans component completely (because of https://github.com/i18next/react-i18next/issues/1014 if you want to know). So I want to disactivate the basic Trans.

Note that if it's not in the default, you would need another setting to turn it off.